package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "buildv1.0": "swagger-cli bundle -t yaml -o docs/fspiop-rest-v1.0-openapi3-snippets.yaml v1.0/openapi3/index.yaml && swagger-cli validate docs/fspiop-rest-v1.0-openapi3-snippets.yaml",
     "buildv1.1": "swagger-cli bundle -t yaml -o docs/fspiop-rest-v1.1-openapi3-snippets.yaml v1.1/openapi3/index.yaml && swagger-cli validate docs/fspiop-rest-v1.1-openapi3-snippets.yaml",
+    "buildThirdparty": "swagger-cli bundle -t yaml -o docs/thirdparty-openapi3-snippets.yaml thirdparty/openapi3/index.yaml && swagger-cli validate docs/thirdparty-openapi3-snippets.yaml",
```

Comment:
would it be better to name the script something like `build:thirdparty`, `build:v1.0`, and `build:v1.1` ?

Reply:
i like the former `build:thirdparty` 
that seems in line re: script format?

Reply:
yep sounds good. changed.